delete.go

```diff
@@ -94,5 +96,23 @@ func delete(containerID string, force bool) error {
 		return err
 	}
 
+	// In order to prevent any file descriptor leak related to cgroups files
+	// that have been previously created, we have to remove them before this
+	// function returns.
+	cgroupsPath := "foo"
+	if err := removeCgroupsPath(cgroupsPath); err != nil {
+		return err
+	}
+
 	return nil
 }
+
+func removeCgroupsPath(cgroupsPath string) error {
```

Comment:
I am planning to implement a PR about fixing the unit testing in clearcontainers/runtime next week. Is that okay for you if the unit testing of these two functions is done in this next PR ?